mobile/library/common/engine_common.cc

```diff
@@ -59,7 +59,7 @@ class ServerLite : public Server::InstanceBase {
 public:
   using Server::InstanceBase::InstanceBase;
   void maybeCreateHeapShrinker() override {}
-  std::unique_ptr<Envoy::Server::OverloadManager> createOverloadManager() override {
+  absl::StatusOr<std::unique_ptr<Envoy::Server::OverloadManager>> createOverloadManager() override {
```

Comment:
Are we trying to use overload manager in envoy mobile? It seems not. Maybe you can remind me the impetus behind removing exceptions here?

Reply:
unfortunately as it's pulled in in e2e tests its "in scope" and it turns out it's helpful to remove exceptions overall (the number of config crashes I've found is distressingly high)